app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
+    if user && !user.activated?
+      user.activate
+      flash[:success] = t "account_activations.edit.activated"
```

Comment:
```suggestion
      flash[:success] = t ".activated"
```
Xài lazy lookup luôn ông, tương tự update cái dưới luôn nha